### PR TITLE
add groovy support

### PR DESCRIPTION
**Pull request by lazywhite**
support for groovy source file

### CODE REVIEW

File: autoload/SingleCompile/templates/groovy.vim [hunk 1]
```diff
@@ -0,0 +1,28 @@
+" Copyright (C) 2010-2017 white Jia
```
Comment (xuhdev) [inline autoload/SingleCompile/templates/groovy.vim]: Copyright notice should be 2017, not 2010-2017.

File: autoload/SingleCompile/templates/groovy.vim [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+" GNU General Public License for more details.
+
+" You should have received a copy of the GNU General Public License
+" along with SingleCompile.  If not, see <http://www.gnu.org/licenses/>.
+
+" check doc/SingleCompile.txt for more information
+
+function! SingleCompile#templates#python#Initialize()
+    call SingleCompile#SetCompilerTemplate('groovy', 'groovy', 'apache groovy compiler',
```
Comment (xuhdev) [inline autoload/SingleCompile/templates/groovy.vim]: `'Apache Groovy compiler'`

**Comment by lazywhite** [comment]
this PR cant work with python source file, close this PR

**Comment by xuhdev** [comment]
What do you mean by "cant workin with python source file"?

**Comment by lazywhite** [comment]
I found after adding groovy support,  support for python is corrupted,  so I should close this PR before solving this problem

**Comment by lazywhite** [comment]
fixed by changing function name

**Comment by xuhdev** [comment]
I'm so sorry that I forgot to submit reviews, so you couldn't read them! Can you reopen again? It's definitely in consideration.

**Pull request closed without merging**